src/libmongoc/tests/unified/entity-map.c

```diff
@@ -322,9 +344,14 @@ entity_client_new (entity_map_t *em, bson_t *bson, bson_error_t *error)
     * has one host. If useMultipleMongoses unspecified, make no assertion.
     */
    if (use_multiple_mongoses != NULL) {
-      if (!test_framework_uri_apply_multi_mongos (
-             uri, *use_multiple_mongoses, error)) {
-         goto done;
+      if (*use_multiple_mongoses && test_framework_is_loadbalanced ()) {
```

Comment:
A good time to meditate on "use_multiple_mongoses" and the nature of plurality. ;-)

Reply:
Haha, I am following the specification's lead with `useMultipleMongoses`. I wouldn't mind `useMultipleMongeese`.